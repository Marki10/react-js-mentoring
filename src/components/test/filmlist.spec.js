import React, { Component } from 'react';
import FilmsList from '../filmslist';
import { connect } from "react-redux";

const ReactTestRenderer = require('react-test-renderer');

describe('Filmlist', () => {
    it('Render filmlist', () => {
        const component = ReactTestRenderer.create(
            connect(<FilmsList />)
        );
        const testTemp = component.toJSON();
        expect(testTemp).toMatchSnapshot();
    })
});