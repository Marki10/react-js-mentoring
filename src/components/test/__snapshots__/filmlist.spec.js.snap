exports[`Filmlist Render filmlist 1`] = `null`;
